lib/l10n_utils/tests/test_template.py

```diff
@@ -125,3 +125,37 @@ def test_render_no_locale(self, django_render, get_lang_path):
         # Note: no .locale on request
         # Should not cause an exception
         render(request, '500.html')
+
+
+@patch.object(env, 'loader', FileSystemLoader(TEMPLATE_DIRS))
+@override_settings(ROOT=ROOT)
+class TestLocaleTemplates(TestCase):
+    def setUp(self):
+        self.client = Client()
```

Comment:
Importing `Client` and doing this is unnecessary. Django `TestCase` class already has a `self.client` available.